### PR TITLE
Adds fonts, and basic responsiveness

### PR DESCRIPTION
**Pull request by kaw2k**
Getting closer to designs. More a few more tweaks then shifting over to hooking up data. 

https://github.com/aptos-labs/create-aptos-dapp/assets/694324/aa4c77ef-e059-49f2-974f-52bba1d5cae7

### CODE REVIEW

**Comment by 0xmaayan** [comment]
seriously, that looks amazing!

File: templates/digital-asset-template/frontend/App.tsx [hunk 1]
```diff
@@ -14,170 +28,192 @@ const COLLECTION_DESCRIPTION = `Step into the enchanting world of "Forest Friend
...
-          )}
-          <p className="body-sm">{COLLECTION_DESCRIPTION}</p>
-          <div className="flex justify-between items-center">
-            <div className="flex gap-4">
-              <Input />
-              <Button>Mint</Button>
-            </div>
+      <main className="flex flex-col gap-10 md:gap-16 mt-6">
+        {/* Hero */}
+        <section className="hero-container flex flex-col md:flex-row gap-6 px-6">
```
Comment (0xmaayan) [inline templates/digital-asset-template/frontend/App.tsx:42]: I think we can get each section into its own components - would be easier for devs to remove/add components

Comment (kaw2k) [reply templates/digital-asset-template/frontend/App.tsx:42]: Yep! I'm keeping everything colocated at the moment, when everything is mostly in place I am going to componentize everything and tidy things up

**Review by 0xmaayan — APPROVED** [review]
LGTM

**Pull request merged**